_licenses/cc-by-sa-4.0.txt

```diff
@@ -0,0 +1,452 @@
+---
+title: Creative Commons Attribution Share Alike 4.0
+nickname: CC-BY-SA-4.0
+source: https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt
+
+description: Similar to <a href='/licenses/cc-by-sa-4.0/'>CC-BY-4.0</a> but requires derivatives be distributed under the same license. Frequently used for media assets and educational materials. A previous version is the default license for Wikipedia and other Wikimedia projects. Not recommended for software.
```

Comment:
or 

> the same or a similar, <a href="https://creativecommons.org/compatiblelicenses/">compatible</a> license.

Thanks for both corrections. If you want to submit a PR that'd be welcome, otherwise I'll fix in next few days.